WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -629,6 +629,21 @@ + (NSArray *)routes
     isSuccessful = [element fb_selectNextOptionWithOffset:offset error:&error];
   } else if ([order isEqualToString:@"previous"]) {
     isSuccessful = [element fb_selectPreviousOptionWithOffset:offset error:&error];
+  } else if (request.arguments[@"value"]) {
+    NSString *value = request.arguments[@"value"];
+    NSString* endValue = element.wdValue;
+    if(element.wdValue == value) {
+      return FBResponseWithOK();
+    }
+    do {
+      [element fb_selectNextOptionWithOffset:offset error:&error];
+      element = [elementCache elementForUUID:(NSString *)request.parameters[@"uuid"]];
+      if([element.wdValue isEqualToString: value]) {
+        return FBResponseWithOK();
+      }
+    } while(![element.wdValue isEqualToString: endValue]);
+    return FBResponseWithStatus([FBCommandStatus invalidArgumentErrorWithMessage:[NSString stringWithFormat:@"The target value cannot be found on the respective Picker Wheel"] traceback:[NSString stringWithFormat:@"%@", NSThread.callStackSymbols]]);
```

Comment:
Please avoid too long strings

Reply:
Sorry but thats how we are expressing other errors also right?

Reply:
okay